dead_apps/clock/js/panels/alarm/active_alarm.js

```diff
@@ -21,7 +21,7 @@ define(function(require) {
       window.location.origin + '/onring.html');
 
     // Handle the system's alarm event.
-    navigator.mozSetMessageHandler('alarm', this.onMozAlarm.bind(this));
+    // navigator.mozSetMessageHandler('alarm', this.onMozAlarm.bind(this));
```

Comment:
Lets create a bug for finding an alternative to Alarms, and add the bug number to this comment